base/twiceprecision.jl

```diff
@@ -536,6 +536,21 @@ function __convertSRL(::Type{StepRangeLen{T,R,S}}, r::AbstractRange{U}) where {T
 end
 
 function sum(r::StepRangeLen)
+    l = length(r)
+    # Compute the contribution of step over all indices.
+    # Indexes on opposite side of r.offset contribute with opposite sign,
+    #    r.step * (sum(1:np) - sum(1:nn))
+    np, nn = l - r.offset, r.offset - 1  # positive, negative
+    # To prevent overflow in sum(1:n), multiply its factors by the step
+    sp, sn = sumpair(np), sumpair(nn)
+    W = widen(Int)
+    Δn = W(sp[1]) * W(sp[2]) - W(sn[1]) * W(sn[2])
+    s = r.step * Δn
+    # Add in contributions of ref
+    ref = r.ref * l
+    convert(eltype(r), s + ref)
+end
+function sum(r::StepRangeLen{<:Any,<:TwicePrecision,<:TwicePrecision}) where {T}
```

Comment:
you have a dangling `where {T}` and this is causing the test failures

Reply:
Thanks again for the reviews!
I removed the `where {T}`, but I did run into a lot of other test failures locally. Most where BLAS related (couldn't load MKL while running the testsuite?), but also `exp2` test failures.

Reply:
Seems to work on CI, and that's what matters for everyone but you :smile:. I always use the default OpenBlas, but I'm aware of why you're particularly interested in MKL.